manifests/r/Ravencoin/RavenCore/4.6.1/Ravencoin.RavenCore.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-608.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Ravencoin.RavenCore
```

```suggestion
PackageIdentifier: RavenProject.RavenCore
```
As winget follows the naming schame `Publisher.PackageName`, it would be more appropriate if PackageIdentifier was modified as such.